dietpi/dietpi-backup

```diff
@@ -324,76 +338,76 @@ However, this check is a rough estimation in reasonable time, thus it could be m
 		fi
 	}
 
+	# $1: Optional restore point from the archive
 	Run_Restore(){
 
+		local DATA='data'
+		(( $1 > 1 )) && DATA="data_$1"
```

Comment:
this line is giving an error on restore now

```
root@DietPi4:~# dietpi-backup -1
/boot/dietpi/dietpi-backup: line 345: ((: > 1 : syntax error: operand expected (error token is "> 1 ")
```

Reply:
Ah true, in case you do not explicitly restore on of the older backups. It is however visual only. I'll mute it.

Reply:
Done: https://github.com/MichaIng/DietPi/commit/fbc02d6